<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShMappingWidget</class>
 <widget class="QWidget" name="ShMappingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>28</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="cbVal">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>121</red>
           <green>125</green>
           <blue>121</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>121</red>
           <green>125</green>
           <blue>121</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>121</red>
           <green>125</green>
           <blue>121</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="minimumContentsLength">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbMap">
     <property name="maximumSize">
      <size>
       <width>44</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <property name="minimumContentsLength">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbMin">
     <property name="text">
      <string>0</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="dsMin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>-339999999999999996123846586046231871488.000000000000000</double>
     </property>
     <property name="maximum">
      <double>339999999999999996123846586046231871488.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbSwitch">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="dsMax">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>-339999999999999996123846586046231871488.000000000000000</double>
     </property>
     <property name="maximum">
      <double>339999999999999996123846586046231871488.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbMax">
     <property name="text">
      <string>0</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbSync">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../glslDevil.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbVal</sender>
   <signal>activated(int)</signal>
   <receiver>ShMappingWidget</receiver>
   <slot>cbValActivated(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMap</sender>
   <signal>activated(int)</signal>
   <receiver>ShMappingWidget</receiver>
   <slot>cbMapActivated(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbMin</sender>
   <signal>clicked()</signal>
   <receiver>ShMappingWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbMax</sender>
   <signal>clicked()</signal>
   <receiver>ShMappingWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dsMin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ShMappingWidget</receiver>
   <slot>updateData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dsMax</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ShMappingWidget</receiver>
   <slot>updateData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbSwitch</sender>
   <signal>clicked()</signal>
   <receiver>ShMappingWidget</receiver>
   <slot>switchBoundaries()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cbValActivated(int)</slot>
  <slot>cbMapActivated(int)</slot>
  <slot>buttonClicked()</slot>
  <slot>updateData()</slot>
  <slot>switchBoundaries()</slot>
 </slots>
</ui>
